iothub/service/src/Common/Amqp/ClientWebSocketTransport.cs

```diff
@@ -219,18 +233,7 @@ private void CancelPendingWrite()
 
         protected override void AbortInternal()
         {
-            if (!_disposed)
```

Comment:
funny that we had all this code, but didn't make the type disposable

Reply:
yeah